core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -740,6 +740,30 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT =
+      new Builder(Name.WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT)
```

Comment:
Similar to `PropertyKey`, the `MetricKey` targets to be viewed by users who may not view Alluxio source codes. Can you help change the MetricKey name and description to be easier to understand?
Suggestion: WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT -> WORKER_BLOCK_REMOVER_TRY_REMOVE_COUNT
WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_SUCCESS_COUNT -> WORKER_BLOCKS_REMOVER_REMOVED_COUNT
will be even better if we can track the try remove and removed in the whole worker not only in AsyncRemover if possible.

Reply:
I've change the property key name and description. 

> will be even better if we can track the try remove and removed in the whole worker not only in AsyncRemover if possible.

Do you mind using another metrics to monitor all try remove and removed blocks count in the whole worker?